docs/spec/crisis/03_tags.md

```diff
@@ -0,0 +1,12 @@
+# Tags
+
+The crisis module emits the following events/tags:
+
+## Handlers
+
+### MsgVerifyInvariance
+
+| Key       | Value                      |
+|-----------|----------------------------|
+| action    | verify\_invariant          |
```

Comment:
Is the backslash intended to be an escape? It isn't needed in Markdown.

Reply:
it was maybe just my editor highlighting is messed

Reply:
(removed the `\`)